src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -28,274 +43,316 @@ function setupTest(
+                test(
+                  'moves focus back to last opened info link when panel is closed',
+                  setupTest(
+                    async page => {
+                      await page.click(
+                        wrapper.findContentRegion().findLink('[data-testid="info-link-1"]').toSelector()
+                      );
+                      const infoLink = wrapper.findContentRegion().findLink('[data-testid="info-link-2"]').toSelector();
+                      await page.click(infoLink);
+                      await page.click(wrapper.findToolsClose().toSelector());
+                      await expect(page.isFocused(infoLink)).resolves.toBe(true);
+                    },
+                    { pageName: 'with-fixed-header-footer', visualRefresh, mobile, appLayoutWidget }
+                  )
+                );
+                test(
+                  'does not move focus back to last opened info link when panel has lost focus - instead focuses tools toggle',
+                  setupTest(
+                    async page => {
+                      const infoLink = wrapper.findContentRegion().findLink('[data-testid="info-link-2"]').toSelector();
+                      await page.click(infoLink);
+                      await page.click(wrapper.findContentRegion().findContainer().toSelector());
+                      await page.click(wrapper.findToolsClose().toSelector());
+                      await expect(page.isFocused(infoLink)).resolves.toBe(false);
+                      await expect(page.isFocused(wrapper.findToolsToggle().toSelector())).resolves.toBe(true);
+                    },
+                    { pageName: 'with-fixed-header-footer', visualRefresh, mobile, appLayoutWidget }
+                  )
+                );
+              }
+            });
+
+            //tests not relevant for mobile, as panel overlays content
+            testIf(!mobile)(
+              'drawers focus toggles between open and close buttons with a narrow drawer',
+              setupTest(
+                async page => {
+                  const triggerSelector = wrapper.findDrawerTriggerById('test-3').toSelector();
+                  await page.click(triggerSelector);
+                  await page.keys('Enter');
+                  await expect(page.isFocused(triggerSelector)).resolves.toBe(true);
+                  await page.keys('Enter');
+                  await expect(page.isFocused(wrapper.findActiveDrawerCloseButton().toSelector())).resolves.toBe(true);
+                  await page.keys('Enter');
+                  await expect(page.isFocused(triggerSelector)).resolves.toBe(true);
+                },
+                { pageName: 'with-drawers', visualRefresh, mobile, appLayoutWidget }
+              )
+            );
+
+            //tests not relevant for mobile, as panel overlays content
+            //TODO set appLayoutWidget to true && resolve issue of toolbar trigger button on narrow toolbars on its own pull request
```

Comment:
This and the test below will be addressed in separate pull request focusing only on that issue.